openawsem/scripts/mm_analyze.py

```diff
@@ -15,6 +15,8 @@
 from openawsem.helperFunctions.myFunctions import *
 
 def analyze(args):
+    print("exiting")
+    raise SystemeExit()
```

Comment:
Change to System Exit

Reply:
sorry, thought I removed that after testing